aries-site/src/examples/templates/page-layouts/anatomy/PageContainerWide.js

```diff
@@ -1,100 +1,64 @@
 import { useContext } from 'react';
-import { Box, Diagram, Stack, Text, ThemeContext } from 'grommet';
-import { LinkPrevious, LinkNext } from 'grommet-icons';
+import { Header, Main, ResponsiveContext } from 'grommet';
 
+import { AppContainer, PageContainer, pageContainer } from '../components';
 import { ContentArea } from './components';
 
-const connections = [
```

Comment:
As discussed offline, wondering if we leave this example "as is" here and move the more interactive sandbox to the conclusion of the page.

Reply:
I think we head in this direction. I think it has clarity in the diagram and will allow us to move faster.

Reply:
Done.